File: core/src/main/java/com/ustadmobile/core/view/AboutView.java
package com.ustadmobile.core.view;

/**
 * Created by mike on 12/27/16.
 */

public interface AboutView extends UstadView {

    String VIEW_NAME = "About";

    void setVersionInfo(String versionInfo);

    void setAboutHTML(String aboutHTML);
}

File: core/src/main/java/com/ustadmobile/core/view/AboutView.kt
package com.ustadmobile.core.view

/**
 * Created by mike on 12/27/16.
 */

interface AboutView : UstadView {

    fun setVersionInfo(versionInfo: String)

    fun setAboutHTML(aboutHTML: String)

    companion object {

        const val VIEW_NAME = "About"
    }
}
